.eslintrc.js

```diff
@@ -221,24 +222,13 @@ module.exports = {
     'react/jsx-curly-brace-presence': 'error',
     'react/jsx-boolean-value': 'error',
     'react/jsx-handler-names': 'error',
-    'react/jsx-key': 'error',
-    'react/jsx-no-duplicate-props': 'error',
-    'react/jsx-no-undef': 'error',
     'react/jsx-pascal-case': 'error',
-    'react/jsx-uses-react': 'error',
-    'react/jsx-uses-vars': 'error',
-    'react/no-deprecated': 'error',
     'react/no-did-mount-set-state': 'error',
     'react/no-did-update-set-state': 'error',
-    'react/no-direct-mutation-state': 'error',
-    'react/no-string-refs': 'error',
-    'react/no-unknown-property': 'error',
     'react/prop-types': 0,
     'react/prefer-es6-class': 'error',
     'react/prefer-stateless-function': 'error',
-    'react/react-in-jsx-scope': 'error',
     'react/self-closing-comp': 'error',
-    'react/display-name': 'warn',
```

Comment:
I removed rules that were already defined in `plugin:react/recommended` and `plugin:react/jsx-runtime` configs